posts/zzz_DO_NOT_EDIT_introducing.../appendix.R

```diff
@@ -0,0 +1,73 @@
+suppressMessages(library(dplyr))
```

Comment:
@manciniedoardo any idea what went wrong here as to why the folder is named zzz_DO_NOT_EDIT_... ?

@rammprasad see https://github.com/pharmaverse/blog/tree/main/posts examples for how the folder is to be named with the date of the blog first so that they order correctly in the repo.

Reply:
/@rossfarrugia actually, the folder name is as expected - this is a recent update to pharmaverseblog that ensures that when the post is merged into main (usually 1 to 2 weeks after its creation) a custom CICD pipeline automatically sets the date on the post to the date of the merge. See the following section of the README:

![image](https://github.com/user-attachments/assets/9c83abbf-9863-4127-bd1f-4fe34c692e18)

Reply:
Very nice! Hadn't spotted that new pipeline, cool 🚀